Cargo.toml

```diff
@@ -5,12 +5,13 @@ authors = [
     "Marius Seritan <github@seritan.net>",
     "Vincent Prouillet <prouillet.vincent@gmail.com>"
 ]
-keywords = ["sass", "ffi"]
+keywords = ["sass", "scss", "ffi"]
 description = "Higher level binding for the Sass library"
 license = "MIT"
 readme = "README.md"
 repository = "https://github.com/compass-rs/sass-rs.git"
 homepage = "https://github.com/compass-rs/sass-rs"
+categories = ["web-programming", "development-tools::build-utils"]
```

Comment:
I'm not sure about the second category, "Utilities for build scripts and other build time steps." sounds like it would contain things to help with Rust build scripts. I might be wrong though, I don't really understand the crates.io categories (nor can I find them on the site)

Reply:
As far as I understand the build-utils category is for things used from `build.rs`, and that's a reasonable place to precompile Sass.